spec/features/accordion_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+describe "Accordion", js: true, type: :view do
+  scenario "There is a page with the accordion rendered" do
+    given_i_visit_a_page_with_the_accordion_component
+    then_the_accordion_loads
+    then_the_accordion__javascript_loads
+    when_i_click_the_first_accordion_section
+    then_the_accordion_opens
+    when_i_click_the_first_accordion_section
+    then_the_accordion_has_data_attributes
```

Comment:
Is it worth having a check for the "Show all sections" button as well?

Reply:
I'm on the fence about what's worth replicating testing wise that's done on the [Design System side](https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/accordion/accordion.test.js) - however I've added a sense check on `25`

Reply:
Good point, we don't need to duplicate tests. The sense check makes sense - thank you!